app/src/sharedTest/java/org/oppia/android/app/options/OptionsFragmentTest.kt

```diff
@@ -239,37 +251,62 @@ class OptionsFragmentTest {
           targetViewId = R.id.reading_text_size_text_view
         )
       ).check(
-        matches(withText("Large"))
+        matches(withText("Medium"))
       )
     }
   }
 
   @Test
-  fun testOptionsFragment_audioLanguage_testOnActivityResult() {
+  fun testOptionsFragment_appLanguageIsDisplayed() {
     launch<OptionsActivity>(
       createOptionActivityIntent(
         internalProfileId = 0,
         isFromNavigationDrawer = true
       )
     ).use {
-      val resultDataIntent = Intent()
-      resultDataIntent.putExtra(MESSAGE_AUDIO_LANGUAGE_ARGUMENT_KEY, "French")
-      val activityResult = ActivityResult(Activity.RESULT_OK, resultDataIntent)
-
-      val activityMonitor = getInstrumentation().addMonitor(
-        AudioLanguageActivity::class.java.name,
-        activityResult,
-        true
+      testCoroutineDispatchers.runCurrent()
+      onView(
+        atPositionOnView(
+          recyclerViewId = R.id.options_recyclerview,
+          position = 1,
+          targetViewId = R.id.app_language_text_view
+        )
+      ).check(
+        matches(withText("English"))
       )
+    }
+  }
 
-      it.onActivity { activity ->
-        activity.startActivityForResult(
-          createAudioLanguageActivityIntent(summaryValue = "Hindi"),
-          REQUEST_CODE_AUDIO_LANGUAGE
+  @Test
+  fun testOptionsFragment_configChange_appLanguageIsDisplayed() {
+    launch<OptionsActivity>(
+      createOptionActivityIntent(
+        internalProfileId = 0,
+        isFromNavigationDrawer = true
+      )
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(isRoot()).perform(orientationLandscape())
```

Comment:
```suggestion
      onView(isRoot()).perform(orientationLandscape())
      testCoroutineDispatchers.runCurrent()
```
For stabilizing  after changing orientation

Reply:
Done.